aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -0,0 +1,85 @@
+
+### Install the IISIntegration NuGet package
+
+In your application, install [Microsoft.AspNetCore.Server.IISIntegration](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.IISIntegration/). This is an interoperability pack that reads environment variables broadcast by ANCM to set up your app. The environment variables provide configuration information such as the port to listen on. 
+
+### Call UseIISIntegration
+
+In your application's `Main` method, call the `UseIISIntegration` extension method on [`WebHostBuilder`](http://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html#Microsoft.AspNetCore.Hosting.WebHostBuilder.md). 
+
+[!code-csharp[](aspnet-core-module/sample/Program.cs?name=snippet_Main&highlight=12)]
+
+The `UseIISIntegration` method looks for environment variables that ANCM sets, and it does nothing if they aren't found. This behavior facilitates scenarios like [developing and testing on MacOS and deploying to a server that runs IIS](../../tutorials/your-first-mac-aspnet.md).  While running on the Mac, Kestrel acts as the web server, but when the app is deployed to the IIS environment, it automatically hooks up to ANCM and IIS.
+
+### Don't call UseUrls
+
+When you use ANCM, don't call the `UseUrls` method, because the `UseIISIntegration` method picks up the port to listen on from environment variables that ANCM sets. When you run the app without IIS, it listens on the default port number at http://localhost:5000.
+
+If you need to set the port number for when you run the app without IIS, call `UseURLs` **before** you call `IISIntegration`.  When you run without IIS, the port number that you provide to `UseUrls` will take effect because `IISIntegration` will do nothing. But when you run with IIS, the port number specified by ANCM will override whatever you passed to `UseUrls`.
+
+### Configure ANCM options in Web.config
+
+Configuration for the ASP.NET Core Module is stored in the *Web.config* file that is located in the application's root folder. Settings in this file point to the startup command and arguments that start your ASP.NET Core app. For sample Web.config code and guidance on configuration options, see [ASP.NET Core Module Configuration Reference](../../hosting/aspnet-core-module.md).
+
+### Run with IIS Express in development
+
+IIS Express can be launched by Visual Studio using the default profile defined by the ASP.NET Core templates. There's no need to run full IIS as a development server since Core apps don't run in-process in IIS. Whether you run with IIS, IIS Express, or directly from the command line, you are running the exact same code and in most cases the exact same execution environment. Running inside of IIS doesn't buy you anything anymore that you can't easily simulate with a command line environment.
```

Comment:
Just remove full IIS if we don't have anything concrete to say. Any differences between IIS Express and IIS are unrelated to ASP.NET Core.